components/x-teaser/__tests__/__snapshots__/snapshots.test.js.snap

```diff
@@ -28,6 +28,7 @@ exports[`x-teaser / snapshots renders a Hero teaser with article data 1`] = `
       className="o-teaser__heading"
     >
       <a
+        aria-label={null}
```

Comment:
nb although these are `null` in the snapshot, most (all?) DOM/HTML renderers will skip this property in the output